packages/shared/forks/ReactFeatureFlags.www-dynamic.js

```diff
@@ -25,6 +25,7 @@ export const replayFailedUnitOfWorkWithInvokeGuardedCallback = __DEV__;
 // to __VARIANT__.
 export const enableTrustedTypesIntegration = false;
 export const warnAboutShorthandPropertyCollision = true;
+export const disableModulePatternComponents = false;
```

Comment:
Why is this hard coded?

Reply:
It's not a big deal in this case but generally we should be testing both versions of any dynamic flag using __VARIANT__

Reply:
I don't want to go around and wrap every existing test that uses them. Should I?

Reply:
I don't think it's that many, right? And we need to update the tests at some point anyway. Though I can see how it could be annoying to have to front load that work.

Reply:
I guess it's good to do this now because otherwise we won't know if we'll break this path for WWW only.

Reply:
This helped find a few loose ends (missing SSR check and missing warning). So thanks for the nudge.